tests/test_pull_request_creator.py

```diff
@@ -244,7 +244,7 @@ def test_outputs_url_on_success(self, print_mock, create_branch_mock, create_pr_
         assert print_mock.call_count == 1
         found_matching_call = False
         for call in print_mock.call_args_list:
-            if 'set-output' in call.args[0]:
+            if '$GITHUB_OUTPUT' in call.args[0]:
```

Comment:
We need to test this change separately. It may break all our routine workflows which are utilizing this script.

Reply:
We can revert this change for now as we only need to update workflows for now.